src/components/split-button/split-button-test.stories.mdx

```diff
@@ -121,3 +121,22 @@ export const SplitButtonStory = ({
     {SplitButtonStory.bind({})}
   </Story>
 </Canvas>
+
+### In a Form
```

Comment:
I appreciate what you're trying to do here but I don't think we really need a story to prove that there's no bug tbh, it doesn't really provide any value to our docs for consumers. I think we can just remove this 👍🏻 

A codesandbox will be enough for QA to test this on

Reply:
Yeah agree with Sam the snapshot will guard against this being removed so the story is overkill

Reply:
That's fine I can get it removed. I figured having something like this in the `test` directory would give QA something to regression test in case this bug were to reappear again. At least if it's in the `test` directory it wouldn't be published to `carbon.sage.com`.

Reply:
A unit test checking that the button always has a type of `button` would be enough for regression here if you wanted to add something